classes/phing/system/io/Win32FileSystem.php

```diff
@@ -630,7 +630,7 @@ public function lister($f)
      */
     private function fixEncoding($strPath)
     {
-        $codepage = 'Windows-' . trim(strstr(setlocale(LC_CTYPE, 0), '.'), '.');
+        $codepage = 'CP' . trim(strstr(setlocale(LC_CTYPE, 0), '.'), '.');
```

Comment:
As shown in the iconv list of encodings https://gist.github.com/hakre/4188459 `Windows-` is the alias for `CP` as mentioned by @mizunashi-mana, there will be no regression to this. But i would suggest changeing `setlocale(LC_CTYPE, 0)` to `setlocale(LC_CTYPE, '')` to get allways the default locale under windows and not an `C` if not defined in settings properly.